src/windows/ClipboardProxy.js

```diff
@@ -52,7 +52,9 @@ module.exports = {
         } catch (e) {
             errorCallback(e);;
```

Comment:
double semicolon

Reply:
that's from the original code, the file is not touched from my modification, but I will fix that though